staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1887,7 +1887,11 @@ type Container struct {
 	Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
 	// Docker image name.
 	// More info: https://kubernetes.io/docs/concepts/containers/images
-	Image string `json:"image" protobuf:"bytes,2,opt,name=image"`
+	// This field was intentionally optional, refer to the following for more info.
```

Comment:
The message here should probably be:

```
This field is optional to allow higher level config management to default or override container images in workload controllers like Deployments and StatefulSets.
```

We generally would not link to issues in our comments.

Reply:
Done